apps/frontend-pwa/src/pages/course/[courseId]/element/[id].tsx

```diff
@@ -53,10 +53,11 @@ function LearningElement({ courseId, id }: Props) {
     if (questionData?.type) {
       if (
         questionData.type === QuestionType.SC ||
```

Comment:
import the QuestionType from @klicker-uzh/graphql/dist/ops to reuse the graphql type instead of our own const (we can gradually get rid of our own const by using the generated types from the backend). use it as QuestionType.Sc etc. (type generation changes .SC to .Sc, but the string stored within is all caps and still works)